src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import users from '../../api/users';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo = ({ todo }: TodoInfoProps) => {
```

Comment:
```suggestion
export const TodoInfo: FC<TodoInfoProps> = ({ todo }) => {
```
This is better typing for react component
pls fix everywhere